### DISCUSSION TITLE
Can I get a Discord link?

### DISCUSSION THREAD

**Discussion by timstarkk** [trpc/trpc#5038] [General]
The one on the website is invalid. Thanks!

**Comment by sachinraja** [comment]
The one on https://trpc.io? It works for me:
https://discord.gg/wzaMgEJkcf

**Reply by timstarkk** [reply]
yeah it says invite invalid. Either expired, or I don't have permission to join

**Reply by sachinraja** [reply]
try this one:
https://discord.gg/trpc-867764511159091230

**Reply by timstarkk** [reply]
OK thanks! clicking the link took me to the invalid message, but copy/pasting it into discord worked for me. Thanks a bunch @sachinraja
